src/test/cypress/support/commands.ts

```diff
@@ -35,8 +35,8 @@ declare global {
             login(credentials: CypressCredentials, url?: string): any;
             logout(): any;
             loginWithGUI(credentials: CypressCredentials): any;
-            getSettled(selector: string, options?: {}): Chainable<Cypress>;
-            reloadUntilFound(selector: string): Chainable<Cypress>;
+            getSettled(selector: string, options?: {}): Chainable<unknown>;
+            reloadUntilFound(selector: string, interval?: number, timeout?: number): Chainable<undefined>;
```

Comment:
You used once `unknown` and once `undefined` here, is this on purpose?

Reply:
They are the types returned by the custom commands, if i change them to something more explicit like Chainable<Interception> the interpreter complains that they dont match.